openslides/assignments/static/js/assignments/site.js

```diff
@@ -527,20 +528,27 @@ angular.module('OpenSlidesApp.assignments.site', [
         };
         // create new ballot
         $scope.createBallot = function () {
-            $scope.activeTab = 0;
             $http.post('/rest/assignments/assignment/' + assignment.id + '/create_poll/').then(
                 function (success) {
                     $scope.alert.show = false;
                     if (assignment.phase === 0) {
                         $scope.updatePhase(1);
                     }
+                    $timeout(function() {
```

Comment:
Is the timeout here really necessary? This should be in the angular context. If so, check line 549, if a timeout is required there, too.